### PR TITLE
Update Japanese UASD

### LINKED ISSUE #4186: Update Japanese UASD copy

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- Updated Japanese UASD
- Archived old one, but not linked to
- Updated the old/new copy docs

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/legal/ubuntu-advantage-service-description/ja](http://0.0.0.0:8001/legal/ubuntu-advantage-service-description/ja)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Compare it to the [copy doc](https://docs.google.com/document/d/1NHQS8MQxUhtAQZplcIFQP39qmrBOCTH508JpcEueSmI/edit#)

## Issue / Card

Fixes #4186

### CODE REVIEW

File: templates/legal/ubuntu-advantage-service-description/ja/index.html [hunk 1]
```diff
@@ -8,1052 +8,945 @@
...
-        <li>スウェーデン</li>
-        <li>台湾</li>
-        <li>トルコ</li>
-        <li>イギリス</li>
-        <li>アメリカ</li>
-      </ul>
-      <p><a href="/legal/ubuntu-advantage-service-description#service-description">上へ戻る</a></p>
+        <ul class="p-list">
+          <li class="p-list__item">開発者用ツール</li>
+          <li class="p-list__item">ベースのUbuntuデスクトップイメージに含まれないパッケージのサポート</li
```
Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/ja/index.html]: Remove `*`

Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/ja/index.html]: Should just be `Rancher 2.x`

Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/ja/index.html]: Should be "Rancher 1.xからRancher 2.xへの作業負荷の移行"

Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/ja/index.html]: The number order here has got lost

Comment (anthonydillon) [inline templates/legal/ubuntu-advantage-service-description/ja/index.html]: There is an incorrect character in this sentence: "1.1 Canonicalはサービスの利用開始時に、お客様の技術担者一名に対して、Landscape、サポートポータルおよびオンラインのナレッジベースへのアクセスを提供します。"

**Comment by pmahnke** [comment]
@anthonydillon thanks, a few mess-ups left in there from textile parsing and other regexs

**Pull request merged**